app/assets/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,74 @@
+.global-nav {
+  background-color: $c-blue;
+  box-shadow: 2px 2px 4px rgba(black, 0.8);
+  padding: 20px;
+  a {
+  color: white;
+  font-family: $f-logo;
+  margin-left: 4%;
+  text-decoration: none;
+  outline-color: orange;
+  @include fluid-size('font-size', 16px, 27px);
+  }
+
+  a.nav-item {
+    float: right;
+    font-family: $f-body;
+    margin-top: 0.7%;
+    @include breakpoint($small-screen) {
+      margin-top: 0.2%;
+    }
+    margin-left: 3%;
+    margin-right: 4%;
+    @include fluid-size('font-size', 10px, 21px);
+
+    &.selected {
+      border-bottom: 5px solid orange;
+    }
+
+    &:hover,
+    &:focus {
+      opacity: 0.6;
+    }
+  }
+}
+
+.global-header
+{
+  position: relative;
+  padding: 4%;
+  padding-top: 7%;
+  padding-bottom: 7%;
```

Comment:
`padding: 7% 4%` for shorthand

When you use only 2 values for padding/margin shorthand, then it applies top/bottom and left/right together. So this short-hand is the same as:
```
padding-top: 7%;
padding-right: 4%;
padding-bottom: 7%;
padding-left: 4%;
```